### PR TITLE
Add topics to repos

### LINKED ISSUE #82: Add topics to aid in repo descoverability
Repos should have the short code / abbreviation for the ministry that owns the repo as one of the `topics`. This will greatly aid in repo discoverability and allow ministries to search for the assets they own.

Some examples of how this would work are: All Attorney General repos would have `AG` as a topic. Likewise, all Citizen Services repos will have `CITZ` as a topic.

### PR DESCRIPTION
**Pull request by jleach**
Topics are a great way to make repos more discoverable. This PR enables the bot to create a issue if a repo is missing the ministry short-code (i.e. CITZ) in the list of topics. Adding this short code will enable ministries to search for and discover what repos belong to them. It will also aid in tracking repo ownership and compliance.

Fixes #82

List of Short Codes

| Short Code | Organization Name |
| :--------- | :------------ |
| AEST | Advanced Education, Skills & Training |
| AGRI | Agriculture |
| ALC | Agriculture Land Commission |
| AG | Attorney General |
| MCF | Children & Family Development |
| CITZ | Citizens' Services |
| DBC | Destination BC |
| EMBC | Emergency Management BC |
| EAO | Environmental Assessment Office |
| EDUC | Education |
| EMPR | Energy, Mines & Petroleum Resources |
| ENV | Environment & Climate Change Strategy |
| FIN | Finance |
| FLNR | Forests, Lands, Natural Resource Operations & Rural Development |
| HLTH | Health |
| FLNR | Indigenous Relations & Reconciliation |
| JEDC | Jobs, Economic Development & Competitiveness |
| LBR | Labour Policy & Legislation |
| LDB | BC Liquor Distribution Branch |
| MMHA | Mental Health & Addictions |
| MAH | Municipal Affairs & Housing |
| BCPC | Pension Corporation |
| PSA | Public Safety & Solicitor General & Emergency B.C. |
| SDPR | Social Development & Poverty Reduction |
| TCA | Tourism, Arts & Culture |
| TRAN | Transportation & Infrastructure |

### CODE REVIEW

File: __tests__/repository.test.ts [hunk 1]
```diff
@@ -48,34 +59,48 @@ jest.mock('../src/libs/utils', () => ({
...
+
         // @ts-ignore
         checkIfRefExists.mockReturnValueOnce(true);
         // @ts-ignore
         hasPullRequestWithTitle.mockReturnValueOnce(true);
         await expect(addLicenseIfRequired(context)).resolves.toBe(undefined);
     });
 
-    it('Adding a license should fail because add file failed', async () => {
+    it.skip('Adding a license should fail because add file failed', async () => {
```
Comment (patricksimonian) [inline __tests__/repository.test.ts:101]: skippedfor any particular reason?

File: templates/why-topics.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+### Why Topic
+
+In short order we'll add our 800th repo. This large number clearly demonstrates the success of using GitHub and our Open Source initiative. This huge success means its critical that we work to make our content as discoverable as possible. Through discoverability, we promote code reuse across a large decentralized organization like the Government of British Columbia.
+
+### What to do
+
+Below is a table of abbreviation a.k.a short codes for each ministry; they're the ones used in all `@gov.bc.ca` email addresses. Please add the short codes of the ministry that "owns" **this** repo as a `topic`:
+
+1. Go to the main page of your repo;
+2. In the upper-left you'll see a `mange topics` link (click or touch it);
```
Comment (patricksimonian) [inline templates/why-topics.md:14]: manage

Comment (jleach) [reply templates/why-topics.md:14]: Fixed.

File: __tests__/repository.test.ts (hunk 1 above)
Comment (jleach) [reply __tests__/repository.test.ts]: Fixed. Was disabled until I had a chance to fix it.

**Pull request merged**